test/AWS.Deploy.CLI.UnitTests/RecommendationTests.cs

```diff
@@ -419,6 +419,52 @@ public async Task IsDisplayable_ManyDependencies()
             Assert.True(fargateRecommendation.IsOptionSettingDisplayable(vpcIdOptionSetting));
         }
 
+        [Fact]
+        public async Task IsDisplayable_NotEmptyOperation()
```

Comment:
Please verify that the server mode tests are properly accepting and handling these new fields

Reply:
I added a test for the new ExistingVPC typehint. The other fields already have tests and no need to redo.